config/routes.rb

```diff
@@ -1,3 +1,14 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root "bathrooms#index"
+
+  namespace :api do
+    
+    namespace :v1 do
+
+      resources :bathrooms
```

Comment:
You only have one action on the API controller, so please use the only keyword to narrow it down to index

Reply:
Also remove spacing between :P